example/netcdf/t8_write_forest_netcdf.cxx

```diff
@@ -13,6 +13,99 @@
 
 T8_EXTERN_C_BEGIN ();
 
+/* Function that times the duration of writing out the netCDF File, given a specific variable storage and access pattern */
+static void
+t8_example_time_netcdf_writing_operation (t8_forest_t forest,
+                                          sc_MPI_Comm comm,
+                                          int netcdf_var_storage_mode,
+                                          int netcdf_var_mpi_access,
+                                          const char *title)
+{
+#if T8_WITH_NETCDF
+  double              start_time, end_time, duration, global;
+  int                 retval;
```

Comment:
I would prefer if we had additional data on the forest as is the case for an application code.

Can we add at lease one integer and one double field here?
Maybe fill with random data or (if we are interested in compression) some function values?